react/week3/todo-list/src/Components/Wrapper.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+function Wrapper() {
+    return (
+        <div>
+            
+        </div>
+    )
+}
+
+export default Wrapper

```

Comment:
I think this component may be unused?

Reply:
No, not yet, was about to upload/add children wrapper.
![image](https://user-images.githubusercontent.com/84173609/143777571-2cd82e35-a6e2-489e-832f-66d6cdec0c3a.png)